### PR TITLE
Merge from Ryan's Development Branch

### PR DESCRIPTION
**Pull request by qu-gg**
Please check if the format is in-line with set standards for other classes and if any other functionality could be added before merge.

File in question: turrets.py

### CODE REVIEW

File: shipyard/models/drives.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def __init__(self, drive_type):
         Drive.__init__(self, drive_type)
 
         # grab additional info from json
-        data = get_file_data(jdrive_data.json)
+        data = get_file_data("jdrive_data.json")
```
Comment (Milkshak3s) [inline shipyard/models/drives.py]: Whoops lol

Blaming you for missing on review

File: shipyard/models/turrets.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+"""
+@file Turrets.py
```
Comment (Milkshak3s) [inline shipyard/models/turrets.py]: lowercase "T"

File: shipyard/models/turrets.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+        # Check if any weapon matches this weapon. If none, print error.
+        for w in self.weapons:
+            if wep == w:
+                self.weapons.remove(w)
+                break
+        print("Error: {} not attached to {}".format(part, self.name))
+
+
+def turret_test():
```
Comment (Milkshak3s) [inline shipyard/models/turrets.py]: Remove print test like this; can put tests in a separate "tests" directory and use a testing framework like pytest: [https://docs.pytest.org/en/latest/](https://docs.pytest.org/en/latest/)

Right now, fine to just remove and I can throw up some example tests on a new PR

**Comment by qu-gg** [comment]
Changes made - requesting re-review

**Pull request merged**